modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_content-type: PROCEDURE
 [id="install-sno-generating-the-discovery-iso-with-the-assisted-installer_{context}"]
-= Generating the discovery ISO with the Assisted Installer
+= Generating the installation ISO with the Assisted Installer
 
 Installing {product-title} on a single node requires a discovery ISO, which the Assisted Installer (AI) can generate with the cluster name, base domain, Secure Shell (SSH) public key, and pull secret.
```

Comment:
Even though these lines already existed - I'm not sure AI is an official acronym for the assisted installer, so perhaps it should be avoided. Also everything after and including "with the..." sounds a bit odd and too verbose. The users should go to the wizard, and the wizard will ask them for all those details. All those details also have nothing to do (at-least as far as the user is concerned) with the ISO file, so it's a bit odd to mention them in the same breath